### ISSUE TITLE
v0.16.0 CLI with XLSX data files, -s and -ct values versus values in xlsx Library tab incorrect precedence? Or, wrong documentation?

### ISSUE THREAD

**Issue by mhungria** [cdisc-org/cdisc-rules-engine#1723]
### Standard

CDISC SDTM/SDTMIG

### Reference Rule ID(s)

CG0312

### Conformance Rule ID(s) (if published in CORE)

CORE-

### JIRA Ticket

https://jira.cdisc.org/projects/CORERULES/issues/CORERULES-

### CLI Command Used (if applicable)

python3 core.py validate -s SDTMIG -v 3.4 -d /path/to/datasets/xyz.xlsx

### Rule YAML

```YAML
See attached
```

### Attach any sample test data file(s)

[CORE-002005-positive-02.xlsx](https://github.com/user-attachments/files/27449062/CORE-002005-positive-02.xlsx)

[CORE-002005,CG0312.yml](https://github.com/user-attachments/files/27449234/CORE-002005.CG0312.yml)

### Attach any output report and/or log file(s)

v0.16.0 CLI processing XLSX data files, -s versus values in xlsx Library tab incorrect precedence? or, wrong documentation?

The CLI command has -s sdtmig -v 3.4
The attached test data has standard: sdtmig 3.2

<img width="164" height="122" alt="Image" src="https://github.com/user-attachments/assets/29a3820d-df98-4907-b564-8c89a9c30ae1" />

And the results indicate:

<img width="909" height="456" alt="Image" src="https://github.com/user-attachments/assets/b3057533-7797-4f2c-8742-a52b69aaa218" />

### Expected output

I would expect the values in the xlsx file will take precedence, unless the documentation indicates a different precedence.

**Comment by eljanssens** [comment]
Hi Marcelina,
Thank you very much for the detailed investigation and for taking the time to report this, much appreciated.
By design, the engine should take into account the IG version specified in the engine command. In a production setting, datasets will not include a library tab, so this type of discrepancy would not occur.
What we are currently seeing is specific to the test data, where there is a mismatch between the IG version in the library tab and the one provided via the engine command. Looking ahead, with the updated CDISC approach, the library tab will no longer be used. Instead, the engine command will effectively be mimicked via an env file.
Given that this situation is limited to the current test setup, and will not occur in future testing environments or in production, we do not see sufficient added value in addressing this nor documenting this temporary situation.
